### DISCUSSION TITLE
Rookie to SeaWeedFS

### DISCUSSION THREAD

**Discussion by XavierLemos** [seaweedfs/seaweedfs#5217] [General]
Hi, I'm doing a university project and I choose to explore SeaWeedFS.
I only started a few days ago, I'm a little lost in the volumes servers. 
I have a master server, and then I need to create 2 new volumes servers but in different machines right?
Now I'm doing the part of writing a File (described in the Readme section) but I'm not understanding how it works.
How do I check if it works, or see if the file is there, or even see it's contents.
Does anyone know a good tutorial/website or any kind of documentation that would help.
Again, I'm new to this software soo I haven't found much information yet about it.
Thank you for the help in advance!
Best regards!

**Comment by chrislusf** [comment]
please go through all contents in https://github.com/seaweedfs/seaweedfs/wiki

**Reply by XavierLemos** [reply]
Hey, sorry for not responding earlier and thank you for the help.
I have another question, when I setup the Filer service on a different machine from the master, it says the service has started and it is running but it's not doing it properly, when I enter the IP and the port in the browser, it does not establish a connection.
If in the master I just do the command "weed filer" it will start a Filer service instantly, but as I said, I want it in a different machine.
What am I doing here wrong or not understanding?
Thank you for the help in advance!
Best regards!

**Reply by guliujian** [reply]
Did you follow the instruction from "https://github.com/seaweedfs/seaweedfs/wiki/Production-Setup#setup-filer" for setup a filer ,provide it with the master server you have set up .
